web-marketplace/src/components/organisms/ProjectDashboardBanner/ProjectDashboardBanner.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLingui } from '@lingui/react';
+import { useMediaQuery, useTheme } from '@mui/material';
+
+import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import OutlinedButton from 'web-components/src/components/buttons/OutlinedButton';
+import ArrowDownIcon from 'web-components/src/components/icons/ArrowDownIcon';
+import ClipboardIcon from 'web-components/src/components/icons/ClipboardIcon';
+import { DraftDocumentIcon } from 'web-components/src/components/icons/DraftDocumentIcon';
+import EditIcon from 'web-components/src/components/icons/EditIcon';
+import EyeIcon from 'web-components/src/components/icons/EyeIcon';
+import { HorizontalDotsIcon } from 'web-components/src/components/icons/HorizontalDotsIcon';
+import TrashIcon from 'web-components/src/components/icons/TrashIcon';
+import ProjectPlaceInfo from 'web-components/src/components/place/ProjectPlaceInfo';
+import { Title } from 'web-components/src/components/typography';
+
+import {
+  CONVERT_TO_DRAFT,
+  DELETE_PROJECT,
+  EDIT,
+  MAX_ADDRESS_LENGTH_DESKTOP,
+  MAX_ADDRESS_LENGTH_MOBILE,
+  MIGRATE_PROJECT,
+  REGISTER_WITH_PROTOCOL,
+  UNTITLED_PROJECT,
+  VIEW,
+} from './ProjectDashboardBanner.constants';
+import { ProjectBannerProps } from './ProjectDashboardBanner.types';
+import { truncateEnd } from './ProjectDashboardBanner.utils';
+
+const ProjectDashboardBanner: React.FC<ProjectBannerProps> = ({
+  project,
+  canEdit,
+}) => {
+  const { _ } = useLingui();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  const navigate = useNavigate();
+
+  const truncatedPlace = truncateEnd(
+    project.place ?? '',
+    isMobile ? MAX_ADDRESS_LENGTH_MOBILE : MAX_ADDRESS_LENGTH_DESKTOP,
+  );
```

Comment:
we could use https://github.com/tailwindlabs/tailwindcss-line-clamp instead
thoughts @r41ph ?

we could use it for the project name too which isn't truncated currently:

<img width="849" height="302" alt="image" src="https://github.com/user-attachments/assets/57791a2e-5621-4b27-abc4-fe707593e787" />

Reply:
It looks to me like it is being handled correctly now.